Kitodo/src/main/java/org/kitodo/production/forms/LanguageForm.java

```diff
@@ -72,11 +73,12 @@ private void setSessionLocaleFieldId() {
         if (Objects.isNull(ServiceManager.getUserService().getAuthenticatedUser())) {
             key = ConfigCore.getParameterOrDefaultValue(ParameterCore.LANGUAGE_DEFAULT);
         } else {
+            Integer id = ServiceManager.getUserService().getAuthenticatedUser().getId();
```

Comment:
Since I don't control Codacy… Can you please spell out `id` as `identifier`? :P

Reply:
Codacy has passed ;)

Reply:
Nice workaround I accidentally produced!